resources/language/resource.language.fr_fr/strings.po

```diff
@@ -310,8 +310,300 @@ msgstr "Démarrer après 5 minutes d'inactivité"
+
+msgctxt "#30125"
+msgid "Purge library"
+msgstr "Vider la bibliothèque"
+
+msgctxt "#30126"
+msgid "Do you really want to purge your library?\nAll exported items will be deleted."
+msgstr "Voulez-vous vraiment vider la bibliothèque ?\nTous les élements exportés seront supprimés."
+
+msgctxt "#30127"
+msgid "Awarded rating of {}"
+msgstr "Note attribuée de {}"
+
+msgctxt "#30128"
+msgid "No contained titles yet..."
+msgstr "Pas encore de titres pour le moment..."
+
+msgctxt "#30129"
+msgid "Enable Up Next integration"
+msgstr "Activer l'intégration Up Next"
+
+msgctxt "#30130"
+msgid "Show notification instead of modal dialog for errors"
+msgstr "Afficher une notification au lieu d'une fenêtre modale pour les erreurs"
+
+msgctxt "#30131"
+msgid "Check the logfile for detailed information"
+msgstr "Vérifier le fichier de log pour plus d'informations"
+
+msgctxt "#30132"
+msgid "Purge in-memory cache"
+msgstr "Vider le cache en mémoire"
+
+msgctxt "#30133"
+msgid "Purge in-memory and on-disk cache"
+msgstr "Vider le cache en mémoire et sur le disque"
+
+msgctxt "#30134"
+msgid "Enable execution timing"
+msgstr "Activer le temps d'excution"
+
+msgctxt "#30135"
+msgid "Cache cleared"
+msgstr "Cache supprimé"
+
+msgctxt "#30136"
+msgid "Enable 1080p Unlock (not recommended for Android)"
+msgstr "Activer déverrouillage 1080p (non recommandé sur Android)"
+
+msgctxt "#30137"
+msgid "Enable VP9 profiles (not use with HEVC, disable if it causes artifacts)"
+msgstr "Activer les profils VP9 (ne pas utiliser avec HEVC, à désactiver si artefacts)"
+
+msgctxt "#30138"
+msgid "The requested action timed out. The background services may not yet be available if you just started Kodi/the addon. In this case, try again in a moment."
+msgstr "L'action n'a pas pu se terminer. Les services en arrière-plan peuvent ne pas être disponbile si vous venez juste de démarrer Kodi ou l'extension. Dans ce cas, essayez ultérieurement."
+
+msgctxt "#30139"
+msgid "Enable IPC over HTTP (use when experiencing AddonSignals timeouts)"
+msgstr "Activer IPC sur HTTP (à utiliser si vous rencontrez des erreurs AddonSignals timeouts)"
+
+msgctxt "#30140"
+msgid "Migrate Library to new format"
+msgstr "Convertir la bibliothèque vers le nouveau format"
+
+msgctxt "#30141"
+msgid "Disable startup notification"
+msgstr "Désactiver la notification au démarrage"
+
+msgctxt "#30142"
+msgid "Custom"
+msgstr "Personnalisé"
+
+msgctxt "#30143"
+msgid "Custom view ID"
+msgstr "Vue par ID personnalisé"
+
+msgctxt "#30144"
+msgid "Disable WebVTT subtitle support"
+msgstr "Désactiver le support sous-titres WebVTT"
+
+msgctxt "#30145"
+msgid "Recently added"
+msgstr "Récement ajouté"
+
+msgctxt "#30146"
+msgid "Discover the latest content added."
+msgstr "Découvrir le contenu récement ajouté."
+
+msgctxt "#30147"
+msgid "Limits list results to about (higher numbers can cause timeouts)"
+msgstr "Limiter la liste de résultat à (un nombre élevé peut causer des erreurs)"
```

Comment:
timeouts is not a generic error

Reply:
We don't have a really nice translation for "timeout" but "temps mort" can make the deal